addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -385,7 +419,7 @@ public String convertStream(HttpsURLConnection connection) throws IOException {
             }
         }
 
-        Scanner inputScanner = StringUtils.isEmpty(charSet) ? new Scanner(readerStream)
+        Scanner inputScanner = StringUtils.isEmpty(charSet) ? new Scanner(readerStream, "UTF-8")
```

Comment:
You may want to use `StandardCharsets.UTF_8.name()`.

Reply:
done, and I have replaced it on all other occurrences.